va_explorer/templatetags/va_explorer_tags.py

```diff
@@ -11,6 +12,27 @@
 
 register = template.Library()
 
+@register.filter
+def replace(value):
+    value = value.strip()
```

Comment:
I might just lowercase `value` and then check on the lowercased version in your conditionals. Then you would catch, "DK", "dk", "Dk", "NaN", "NAN", "REF", "ref" etc.

Reply:
I don't think that would work because this would lowercase all value's, because my `else` condition is to just return whatever input was received.

Reply:
`lower_value = value.lower() `and use `lower_value` that in your conditionals, then return the original `value`.